man/man1/kbsecret-cp.1.ronnpp

```diff
@@ -0,0 +1,26 @@
+kbsecret-cp(1) - copy kbsecret(1) record(s)
+===========================================
+
+## SYNOPSIS
+
+`kbsecret cp` <src_sess> <dst_sess> [options] <record ![record ...]>
+
+## DESCRIPTION
+
+`kbsecret cp` copies the requested record(s) between sessions
+
+## OPTIONS
+
+* `-m`, `--move`:
+	Delete the record(s) after copying.
+
+## EXAMPLES
+
+```
+	$ kbsecret cp home shared website_login
+	$ kbsecret cp -m sandbox production api_key
+```
+
+## SEE ALSO
+
+kbsecret(1), kbsecret-rm(1)
```

Comment:
I'd add links to `kbsecret-new(1)` and `kbsecret-session(1)` as well, and cross-link in each of these manual pages as well.

Also, make sure to update the list of commands in `kbsecret(1)` to include this one (alphabetized).